packages/sync-actions/test/cart-discounts-sync.spec.js

```diff
@@ -0,0 +1,289 @@
+import cartDiscountsSyncFn, {
+  actionGroups,
+} from '../src/cart-discounts'
+import { baseActionsList } from '../src/cart-discounts-actions'
+
+describe('Exports', () => {
```

Comment:
Can we prefix this with `Cart Discounts` so tests suites are well described?

Also [here](https://github.com/commercetools/nodejs/pull/286/files#diff-ad3fd9838b0f01608285f49c706da4d7R27)